README.md

```diff
@@ -19,17 +19,29 @@ Limitations:
 
 ## Configuration
 
-Under skill settings you can tweak some parameters for chatGPT
-
+Under skill settings you can tweak some parameters for chatGPT.
 
 - `key` - your api_key to access OpenAI
 - `persona` - can be used to create a "persona", give a personality to chatGPT
-- `model` - LLM model to use, eg `"gpt-3.5-turbo", see all options [here](https://platform.openai.com/docs/models)
+- `model` - LLM model to use, eg `gpt-3.5-turbo`, see all options [here](https://platform.openai.com/docs/models)
 
 The default persona is `helpful, creative, clever, and very friendly.`
 
+```shell
+mkdir -p ~/.config/mycroft/skills/skill-ovos-fallback-chatgpt.openvoiceos
+cat <<EOF>~/.config/mycroft/skills/skill-ovos-fallback-chatgpt.openvoiceos/settings.json
+{
+  "key": "sk-XXXYYYZZZAAABBB123",
+  "model": "gpt-3.5-turbo",
+  "persona": "You are a helpful voice assistant with a friendly tone and fun sense of humor",
+  "__mycroft_skill_firstrun": false
+}
+EOF
+```
+
+If you are already running the [skill-ovos-fallback-unknown](https://github.com/OpenVoiceOS/skill-ovos-fallback-unknown) skill, you might have to either uninstall it or change the fallback priority in `mycroft.conf`.
```

Comment:
this should not be needed, the async change below makes this fallback no longer timeout